src/themes/default/elements/icon.variables

```diff
@@ -7,6 +7,7 @@
 ---------------*/
 
 /* Solid Icons */
+@importIcons: true;
```

Comment:
Forgot to add `@imporBrandIcons` ?

Reply:
No, `@importBrandIcons` is already declared here
https://github.com/fomantic/Fomantic-UI/blob/2580bc67fd3e6865ee4dc99c83308c525ef559f7/src/themes/default/elements/icon.variables#L36

Reply:
I see